openapi-spec/swagger.yaml

```diff
@@ -813,7 +814,7 @@ paths:
           description: Forbidden
         '500':
           $ref: '#/responses/HTTPStatus500'
-  '/v1beta/{projectId}/block/volumeGroup/{volumeGroupId}':
+  '/v1beta/{projectId}/block/volumeGroups/{volumeGroupId}':
```

Comment:
This is better changed as a separate PR.  
It's ok to leave it here now.

Reply:
@lasshpanzer Please submit another PR to fix it.